src/transforms/select.d.ts

```diff
@@ -13,16 +13,50 @@ export interface SelectOptions {
   z?: ChannelValue;
 }
 
+/**
+ * Selects points from each series (usually defined by *stroke*, *fill* or *z*)
```

Comment:
The “usually defined by *stroke*, *fill* or *z*” feels unnecessarily imprecise. How would the user know when or not the “usual” definition applies? We should list these channels in order of precedence: *z*, *fill*, *stroke*. The README is more definite:

> The data are grouped into series using the z, fill, or stroke channel in the same fashion as the [area](https://github.com/observablehq/plot/blob/main/README.md#area) and [line](https://github.com/observablehq/plot/blob/main/README.md#line) marks.

And similarly for the area mark:

> By default, the data is assumed to represent a single series (i.e., a single value that varies over time). If the z channel is specified, data is grouped by z to form separate series. Typically z is a categorical value such as a series name. If z is not specified, it defaults to fill if a channel, or stroke if a channel.